src/vm/vars.hpp

```diff
@@ -341,7 +341,7 @@ class REF : public OBJECTREF
 
 };
 
-#define VALIDATEOBJECTREF(objref) ((objref).Validate())
+#define VALIDATEOBJECTREF(objref) (if (objref != NULL) (objref).Validate())
 #define VALIDATEOBJECT(obj) obj->Validate()
```

Comment:
I don't believe I called VALIDATEOBJECT previously, but I will make the change. I am not sure of the full intent of these w.r.t. NULL, but I assume that NULL is ok as I've seen calling code for the 'ref' functionality which explicitly passes in NULL